war/src/main/js/config-scrollspy.less

```diff
@@ -11,6 +11,7 @@
  * Widget styles
  */
 @import "widgets/config/tabbar";
+@import "widgets/add/addform";
```

Comment:
`config-scrollspy` is for the config pages, which are not the same thing as the Add Item page. I suggest creating a specific `add-item-scrollspy.less` and get it to import what's needed.

These changes might drive further modularization of some of the LESS rules in other files.